test/ReactWrapper-spec.jsx

```diff
@@ -3267,6 +3267,29 @@ describeWithDOM('mount', () => {
     });
   });
 
+  describe('.getDOMNode', () => {
+    class Test extends React.Component {
+      render() {
+        return (
+          <div className="outer">
+            <div className="inner">
+              <span />
+              <span />
+            </div>
+          </div>
+        );
+      }
+    }
+    it('should return the outer most DOMComponent of the wrapper', () => {
+      const wrapper = mount(<Test />);
+      expect(wrapper.getDOMNode()).to.have.property('className', 'outer');
+    });
```

Comment:
let's also add a test that grabs the inner DOM node, so we have one test for "root", one for "not root", and one for "throws on multiples"